.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

```diff
@@ -1,60 +0,0 @@
----
-
-name: New Bug Report
-about: Report a problem with the Mu editor.
-
----
-# Bug Report
-
-<!-- Please answer each question below to the best of your ability. It's okay to leave some blank if it doesn't apply to your problem. -->
-What were you trying to do?
----------------------------
-
-
-What steps did you take to trigger the issue?
----------------------------------------------
-
-
-What did you expect to happen?
-------------------------------
-
-
-What actually happened?
------------------------
-
-
-Why is this difference problematic? (it might not be a bug!)
-----------------------------------------------------------
```

Comment:
The issue in the form format looks a much bigger, so I removed this question with the aim to reduce the length of the form a little bit.